openslides_backend/action/actions/user/create.py

```diff
@@ -75,6 +75,8 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
                 if not (instance.get("first_name") or instance.get("last_name")):
                     raise ActionException("Need username or first_name or last_name")
                 instance["username"] = self.generate_username(instance)
+        elif " " in instance["username"]:
```

Comment:
Then you should look with a regex for \W. That's how the generate_username limits the allowed signs and IMO would be more convenient.

Reply:
done